R/class-workbook-wrappers.R

```diff
@@ -3225,21 +3303,23 @@ wb_add_sparklines <- function(wb, sheet = current_sheet(), sparklines) {
   wb$clone(deep = TRUE)$add_sparklines(sheet, sparklines)
 }
 
-#' Ignore error on worksheet
+#' Ignore error types on worksheet
 #'
-#' @param wb workbook
-#' @param sheet sheet
-#' @param dims dims
+#' This function allows to hide / ignore certain types of errors shown in a worksheet.
+#' @param wb A workbook
+#' @param sheet A sheet name or index.
+#' @param dims Cell range to ignore the error
 #' @param calculated_column calculatedColumn
 #' @param empty_cell_reference emptyCellReference
 #' @param eval_error evalError
 #' @param formula formula
 #' @param formula_range formulaRange
 #' @param list_data_validation listDataValidation
-#' @param number_stored_as_text numberStoredAsText
+#' @param number_stored_as_text If `TRUE`, will not display the error if numbers are stored as text.
```

Comment:
This should be true for each of these, but I was obviously to lazy to test all cases 😓